### PR TITLE
fmjpeg2koj recipe

### PR DESCRIPTION
**Pull request by sameeul**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sameeul** [comment]
@conda-forge-admin, please ping conda-forge/help-c-cpp
@conda-forge//help-c-cpp, ready for review!

**Comment by sameeul** [comment]
@conda-forge//help-c-cpp, ready for review!

**Comment by sameeul** [comment]
@conda-forge/staged-recipes: Ready for review

File: recipes/fmjpeg2koj/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+mkdir build
+cd build
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=${PREFIX} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DFMJPEG2K=${PREFIX} ..
+
+cmake --build . --target install --parallel
```
Comment (ocefpaf) [inline recipes/fmjpeg2koj/build.sh:14]: Is there a `make check` or `ctest` to run here?

Comment (sameeul) [reply recipes/fmjpeg2koj/build.sh:14]: The source repo does not have any unit test. So, I just decided to check the existence of some headers and libraries once things are installed. Here is an example build script from the source repo: https://github.com/DraconPern/fmjpeg2koj/blob/master/build.sh

Does that answer your question?

**Pull request merged**